cmd/chart-repo/utils_test.go

```diff
@@ -293,7 +314,7 @@ func Test_fetchAndImportIcon(t *testing.T) {
 
 func Test_fetchAndImportFiles(t *testing.T) {
 	index, _ := parseRepoIndex([]byte(validRepoIndexYAML))
-	charts := chartsFromIndex(index, repo{Name: "test", URL: "http://testrepo.com"})
+	charts := chartsFromIndex(index, repo{Name: "test", URL: "http://testrepo.com", AccessToken: "Bearer ThisSecretAccessTokenAuthenticatesTheClient"})
```

Comment:
I think we should do a specific test for the authenticatedTarballClient, just so that we're testing both scenarios (this currently is only testing the authed scenario)

Reply:
done!